### PR TITLE
Question component

### LINKED ISSUE #105: [FrontEnd] Add question component to Collaboration Page

### PR DESCRIPTION
**Pull request by thanwinnikki**
Fixes #105

### CODE REVIEW

File: frontend/src/components/QuestionPane/index.js [hunk 1]
```diff
@@ -1,7 +1,22 @@
-import React from "react";
+import { Container } from "@mui/material";
```
Comment (Ziyang-98) [inline frontend/src/components/QuestionPane/index.js:1]: Should remove unused imports

File: frontend/src/hooks/useQuestion.js [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
 const useQuestion = (handleOpenNotification) => {
   // Feel free to update this to useState
-  const questionObject = null;
+  const [questionObject, setQuestionObject] = useState("Loading question...");
+
+  const [questionName, setQuestionName] = useState("");
+
+  const search = useLocation().search;
+  const roomId = new URLSearchParams(search).get("roomId");
+  console.log(roomId);
```
Comment (Ziyang-98) [inline frontend/src/hooks/useQuestion.js:13]: should remove console.logs thats for dev purpose when PRing to main

**Review by Ziyang-98 — COMMENTED** [review]
Can help check why CI for matching service started to fail? Thanks

**Comment by lyeyixian** [comment]
> Can help check why CI for matching service started to fail? Thanks

Yea, it failed because the test didn't clean up the db properly. I notice this at other PR also. I will need to change the unit test to use local db

**Review by lyeyixian — APPROVED** [review]
LGTM

**Pull request merged**